package.json

```diff
@@ -12,6 +17,9 @@
   "bugs": {
     "url": "https://github.com/orbitdb/orbit-electron/issues"
   },
+  "homepage": {
+    "url": "https://github.com/orbitdb/orbit-electron/issues"
```

Comment:
I don't think this should point to /issues.

Reply:
It shouldn't and doesn't (now) :)